### PR TITLE
Feature file for population-types API root endpoint

### PR DESCRIPTION
**Pull request by goofballLogic**
### What

Added (failing) component test for population types endpoint

### How to review

1. Verify the feature added makes sense from a high-level reading.
2. Check fixes in main_test

### Who can review

gophers

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -43,6 +45,10 @@ func (f *ComponentTest) InitializeTestSuite(ctx *godog.TestSuiteContext) {
 
 func TestComponent(t *testing.T) {
 	if *componentFlag {
+		// discarding production logging to obtain cleaner reporting of component specifications and results
```
Comment (redhug1) [inline main_test.go:48]: Please remove these lines as log output is required.
You are thinking about form over function.
If in the future someone makes a change in the code and it breaks things, then they will need to see logs.
Reports are of no real consequence, people when checking a feature test run are only looking at the final code.
The final output at the end of all of the feature tests that looks similar to a Zero error code of:
ERRO[0065] 0
Is all that matters.

File: features/population-types.feature [hunk 2]
```diff
@@ -11,4 +11,4 @@ Feature: Population types endpoint
   Scenario: If the root population-types endpoint fails, it should return correct errors
     Given cantabular is unresponsive
     When I access the root population types endpoint
-    Then the service responds with an internal server error saying "nope"
\ No newline at end of file
+    Then the service responds with an internal server error saying "failed to fetch population types"
```
Comment (redhug1) [inline features/population-types.feature:14]: github is looking for a carriage return at end of line 14

**Review by redhug1 — CHANGES_REQUESTED** [review]
a few changes needed.

File: features/population-types.feature (hunk 2 above)
Comment (goofballLogic) [reply features/population-types.feature]: thanks!

File: main_test.go (hunk 1 above)
Comment (goofballLogic) [reply main_test.go]: Added a flag to make the logging output optional during test runs

**Review by redhug1 — APPROVED** [review]
LGTM

**Pull request merged**